apps/api/src/common/constants/notifications.ts

```diff
@@ -53,6 +54,10 @@ export const ProviderDeliveryStatus = {
     SUCCESS_STATES: ['DELIVRD'],
     FAILURE_STATES: ['EXPIRED', 'UNDELIV', 'FAILED'],
   },
+  VC_TWILIO: {
+    SUCCESS_STATES: ['completed'],
+    FAILURE_STATES: ['queued', 'ringing', 'in-progress', 'canceled', 'busy', 'no-answer', 'failed'],
```

Comment:
Are `'queued', 'ringing', 'in-progress'` really failure states? I would assume it means the request is still being processed/in progress.